health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -0,0 +1,399 @@
+                type: "boundaryManagement",
+                hierarchyType: hierarchyType,
+                campaignId: "default"
+            }
+        });
+        return res;
+
+    }
+    const downloadExcelTemplate = async() => {
+        const res = await generateFile()
+        const resFile = await generateTemplate();
+        if (resFile && resFile?.GeneratedResource?.[0]?.fileStoreid) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)